run_iwyu_tests.py

```diff
@@ -183,7 +183,6 @@ def setUp(self):
       'range_for.cc': ['.'],
       're_fwd_decl.cc': ['.'],
       'redecls.cc': ['.'],
-      'relative_include_of_export.cc': ['.'],
```

Comment:
Now it feels like we lost test coverage of the `-I .` vs no header search path...?

Reply:
We did.  I think I decided that the version of the test with a `-I` option was mostly redundant.